### PR TITLE
Lfa 1196 add unit test and config

### PR DESCRIPTION
**Pull request by mwestacott**

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  roots: [
+    "<rootDir>"
```
Comment (markpit) [inline jest.config.js]: rootdir not defined?

File: test/routes/index.spec.unit.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import app from "../../src/app";
+import * as request from "supertest";
+
+describe("Basic URL Tests", () => {
+
+  it("should find start page", async () => {
+    const response = await request(app)
+      .get("/promise-to-file");
+    expect(response.status).toEqual(200);
```
Comment (markpit) [inline test/routes/index.spec.unit.ts]: how do you know it's returning the correct page?  can you check page title or some identifying string in response.text (i think)

File: jest.config.js (hunk 1 above)
Comment (markpit) [reply jest.config.js]: ignore this, <rootDir> apparently points to the dir that contains package.json

Comment (mwestacott) [reply jest.config.js]: Copied from SDK Node and extensions projects. Don't know what this does but will try removing it.

File: test/routes/index.spec.unit.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  });
+
+  it("should find company number page", async () => {
+    const response = await request(app)
+      .get("/promise-to-file/company-number")
+      .set("Referer", "/")
```
Comment (markpit) [inline test/routes/index.spec.unit.ts]: Referer not used at this point. Might not be needed if we can find a way of capturing the page being requested via middleware for the back button.

Comment (mwestacott) [reply test/routes/index.spec.unit.ts]: Will remove 'Refererer' as not yet used or needed by this skeleton app.

**Pull request merged**